src/Microsoft.Azure.WebJobs.Host/Bindings/Path/BindingTemplate.cs

```diff
@@ -84,54 +87,54 @@ public static BindingTemplate FromString(string input, bool ignoreCase = false)
         /// <returns>Resolved string if succeeded.</returns>
         /// <exception cref="InvalidOperationException">Thrown when required parameter value is not available.
         /// </exception>
-        public string Bind(IReadOnlyDictionary<string, string> parameters)
+        public string Bind(IReadOnlyDictionary<string, object> parameters)
```

Comment:
Of course, it doesn't help that the extension methods are in a different namespace :|


---
In reply to: [112509482](https://github.com/Azure/azure-webjobs-sdk/pull/1117#discussion_r112509482) [](ancestors = 112509482,112491637)